content/en/docs/tutorials/verify.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Verifying container image signature with Kubernetes admission tools"
+description: "Verifying container image integrity and signature before deploying to Kubernetes Cluster with Ratify and Kyverno"
+type: docs
+weight: 2
+---
+Digital signatures offer a way for software consumers to verify that the downloaded container image is from a trusted source and has not been tampered with. This verification process is essential for ensuring the security and integrity of an image before using it. [Ratify](https://github.com/deislabs/ratify) and [Kyverno](https://github.com/kyverno/kyverno/) support verifying container images before deploying them into a Kubernetes cluster to protect against potential security risks. For more information on signing and verifying container images, please refer to [Sign and verify an image with Notation, Ratify, and OPA Gatekeeper](https://ratify.dev/blog/sign-and-verify-image-with-notation-ratify) and [Verifying Image Signatures](https://kyverno.io/docs/writing-policies/verify-images/notary/#verifying-image-signatures).
+
+## Verify image signature with Notation CLI
+ [Verify the image signature as a software consumer](https://ratify.dev/blog/sign-and-verify-image-with-notation-ratify/#verify-the-image-signature-as-a-software-consumer) with the Notation CLI before using it.
+
+``` 
+notation verify $IMAGE
+```
+You can also check the signature digest and inspect the signature and its certificate information to make sure the image is produced by a trusted identity.
+
+```
+notation inspect $IMAGE
+```
+
+## Verifying a container image with Ratify 
+As a prerequisite, you’ll need:
+- Kubernetes v1.20 or higher (You can use [minikube](https://minikube.sigs.k8s.io/docs/start/) if you are new to Kubernetes)
+- Helm v3
+
+Using Ratify will ensure that only signed images are deployed to Kubernetes.
+Set up Gatekeeper with [external data.](https://open-policy-agent.github.io/gatekeeper/website/docs/externaldata)
+
+```console
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+helm install gatekeeper/gatekeeper  \
+   --name-template=gatekeeper \
+   --namespace gatekeeper-system --create-namespace \
+   --set enableExternalData=true \
+   --set validatingWebhookTimeoutSeconds=5 \
+   --set mutatingWebhookTimeoutSeconds=2
+```
+Install the lastest version of Ratify. Specify the certificate generated by Notation for verification purposes.
```

Comment:
You need to give more context on what certificate needs to be used and how to find it. You also need to make the certificate name a variable in the command or let the user know they need to change the name when running the command. Otherwise, users will copy/paste and receive and error without any context for why they have it.

Reply:
I'm unsure why this cert ` --set-file notaryCert=ghcr-networks.io.crt` was used too, as I keep getting an error. @FeynmanZhou could you please help?